src/components/settings/ReportsTab.vue

```diff
@@ -66,6 +118,23 @@ export default {
     handleConfig() {
       this.$emit("submit-config", this.config);
     },
+
+    async openImage() {
+      if (this.$isElectron) {
+        const { status, message } = await this.$electronService.openImage();
+
+        if (status === STATUSES.ERROR) {
+          this.$root.$emit("set-snackbar", message);
+        } else {
+          // const currentPath = this.$router.history.current.path;
```

Comment:
This function doesn’t seem to do anything when successful?

we don’t need a new ipc function to open images, just use a the `src` attribute with a `file://` path like we do [everywhere else](https://github.com/yatt-ai/yattie/blob/3a1f1e12cd96ce3be7d3482caa06fb62391c652f/src/views/PrintView.vue#L113).

Reply:
I have added the action when successful.